.github/workflows/test-and-build.yml

```diff
@@ -63,9 +62,50 @@ jobs:
           QT_QPA_PLATFORM: offscreen
 
 
-  build:
+  # TODO: Consider extracting everything below this line to a separate workflow
+  # which is triggered by pushes to the default branch and GitHub releases.
+  build-and-release-image:
     runs-on: "ubuntu-latest"
     needs: ["test"]
+    if: "github.ref_type == 'tag' || github.ref_type == 'branch'"
```

Comment:
- [ ] I think we can get rid of this conditional on the whole job... we want to run the build as a test on PRs no matter what. We just want to prevent the _push_.

Reply:
62391c7b